### DISCUSSION TITLE
Documentation: "AUC calculation" page

### DISCUSSION THREAD

**Discussion by DBartlettHP** [humanpred/pknca#325] [General]
In the sentence:
"In the figure below, AUC_{0-last} integrate the shaded region."

Should "integrate" be plural?

---
There is a typo within the "AUC to infinity" section.

In the sentence:
"The differences between the tow figures is most evident at time=8 where there is a discontinuity ..."

"tow" should be "two"

**Comment by billdenney** [comment]
Yes, both of these should be updated. Can you please make a PR for both of them?
